### DISCUSSION TITLE
35bffd94-7a41-4cfa-812c-b8aaf148604a

### DISCUSSION THREAD

**Discussion by giscus** [ChooSeoyeon/blog-comment#4] [General]
# 35bffd94-7a41-4cfa-812c-b8aaf148604a

1. 서론

https://choo.oopy.io/35bffd94-7a41-4cfa-812c-b8aaf148604a

**Comment by DebugMind** [comment]
감사합니다 :) 블로그 글 잘 보고 있습니다.

**Comment by POTatO-25** [comment]
안녕하세요! 블로그 덕분에 이미지 업로드 문제 해결했어요 ㅎㅎ 감사합니다 :) 혹시 버킷 생성하는 과정을 제 밸로그에 출처 남기고 사용해도 괜찮을까요?

**Reply by ChooSeoyeon** [reply]
도움이 되셨다니 다행이네요 ㅎㅎ 물론이죠~
